model-optimizer/extensions/front/tf/TFResizeToInterpolate.py

```diff
@@ -32,13 +31,9 @@ def replace_tf_resize(graph: Graph, resize: Node, interpolation_mode: str):
 
     shape = Shape(graph, {'name': resize_name + '/shapeof'}).create_node()
 
-    layout = graph.graph['layout']
-    height_dim = get_height_dim(layout, 4)
-    width_dim = get_width_dim(layout, 4)
-
     ss = create_op_with_const_inputs(graph, StridedSlice,
-                                     {1: int64_array([height_dim]),
-                                      2: int64_array([width_dim + 1]),
+                                     {1: int64_array([1]),
```

Comment:
Because the TFResize operation has a fixed input tensor layout NHWC (refer to the documentation https://www.tensorflow.org/api_docs/python/tf/image/resize). So in order to calculate output shape we need to use just this layout.